alacritty/src/renderer/text/builtin_font.rs

```diff
@@ -1,5 +1,7 @@
-//! Hand-rolled drawing of unicode [box drawing](http://www.unicode.org/charts/PDF/U2500.pdf)
-//! and [block elements](https://www.unicode.org/charts/PDF/U2580.pdf), and also powerline symbols.
+//! Hand-rolled drawing of unicode [box drawing](http://www.unicode.org/charts/PDF/U2500.pdf),
+//! [block elements](https://www.unicode.org/charts/PDF/U2580.pdf)
+//! and [symbols for legacy computing](https://www.unicode.org/charts/PDF/U1FB00.pdf),
+//! and also powerline symbols.
```

Comment:
As stated in my last review:

> This comment should be reworded to not include what exactly is being drawn. Otherwise we'll have to change it constantly.

I'd appreciate not resolving reviews that haven't been completed.

Reply:
Oh, I see what you meant by that review.

Does this rephrase work:
```rust
//! Hand-rolled drawing of unicode characters that need to fully cover their character area.
```